src/renderer/Layout/GridLayout/renderer.css

```diff
@@ -50,6 +50,7 @@
 .grid-layout .grid-layout-row .grid-block .widget-weather {
   background-color: #6495ED;
   padding: 20px;
+  overflow: auto;
```

Comment:
The `overflow` property specifies what should happen if content overflows an element's box. [(Reference)](https://www.w3schools.com/cssref/pr_pos_overflow.asp)

The Weather widget should be 100% height of its container.